packages/board-server/Dockerfile

```diff
@@ -1,11 +1,34 @@
-FROM node:20
-COPY --from=breadboard . /breadboard/
-# For the time being, we will assume container deployments are exclusively sqlite.
-ENV STORAGE_BACKEND "sqlite"
-ENV GOOGLE_CLOUD_PROJECT "n/a"
-ENV ALLOWED_ORIGINS ""
-WORKDIR /breadboard
+# Build stage
+FROM node:20 AS build
+WORKDIR /build
+
+# Copy the entire monorepo
+COPY --from=breadboard / ./
```

Comment:
Just an idea for the future: It can be nice to do a selective copy of just the `package.json` and `package-lock.json` initially, and then copy the rest after `npm ci`. That way the `npm ci` layer is not affected by source changes, and we'll get more cache hits during build.

Reply:
I actually was doing this before but I ran into persistent errors like this in symlinked dependencies. 

```
tsconfig.json:13:14 - error TS6053: File '@google-labs/tsconfig/base.json' not found.
3.897 
3.897 13   "extends": "@google-labs/tsconfig/base.json",
3.897       
```